apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -179,6 +193,20 @@
 
 		<!-- loading indicator to replace the menu -->
 		<div v-else class="icon-loading-small sharing-entry__loading" />
+
+		<!-- Modal to open whenever we have a QR code -->
+		<NcDialog size="normal"
+			:open.sync="showQRCode"
+			:name="title"
+			:closeOnClickOutside="true"
+			@close="showQRCode = false">
+			<div class="sharing-entry__qrCode">
+				<qrcode tag="img"
+				:value="shareLink"
+				class="sharing-entry__qrCode__tag">
+				</qrcode>
```

Comment:
Locally registered should also work.

```suggestion
				<VueQrcode tag="img"
					:value="shareLink"
					class="sharing-entry__qrCode__tag">
				</VueQrcode>
```